extensions/azure-extensions/src/main/java/io/druid/firehose/azure/AzureFirehoseDruidModule.java

```diff
@@ -0,0 +1,25 @@
+package io.druid.firehose.azure;
+
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.jsontype.NamedType;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
+import io.druid.initialization.DruidModule;
+
+import java.util.List;
+
+public class AzureFirehoseDruidModule implements DruidModule {
```

Comment:
can we have this as a part of the azure module versus having a separate module just for the firehose?